### DISCUSSION TITLE
Search in an area of the map for all visits there (no matter the date)

### DISCUSSION THREAD

**Discussion by Carunga** [Freika/dawarich#319] [Ideas]
Hello,
first and foremost: great project, for me it is very usable and helpful as is. The GPX-import was able to handle a large collection of files and I love the watch-folder!
One thing I would like to do sometimes is to search in a certain area for all visits I have done there. This is i.e. helpful for remembering the details a mountain bike  trail you rode 5 years ago and maybe want to recommend it to a friend.
Is there a way to do that? In the UI I will always have to limit the search for a year, no?
Thanks!

**Comment by Freika** [comment]
I'll move it to Ideas category to implement it in the future, so it won't be lost, thanks for suggestion!

**Comment by Freika** [comment]
Done in 0.25.0
